server/src/controllers/notification.controller.ts

```diff
@@ -0,0 +1,18 @@
+import { Controller, HttpCode, HttpStatus, Post } from '@nestjs/common';
+import { ApiTags } from '@nestjs/swagger';
+import { AuthDto } from 'src/dtos/auth.dto';
+import { Auth, Authenticated } from 'src/middleware/auth.guard';
+import { NotificationService } from 'src/services/notification.service';
+
+@ApiTags('Notification')
+@Controller('notification')
```

Comment:
Probably this should be `notifications`, as we consistently switched to plural endpoints

Reply:
Yep, don't make Jason angry :laughing: